src/server/server_family.cc

```diff
@@ -68,19 +68,26 @@ struct ReplicaOfFlag {
   }
 };
 
+struct CronExprFlag {
+  static inline const std::string cronPrefix{"0 "};
```

Comment:
It's slightly better to define it as
```cpp
constexpr std::string_view kCronPrefix = "0 "sv;
```

This way it doesn't have a lifetime, and is always available. Also, note the kCamelCase convention that we use for constants